src/main/java/ChatBot.java

```diff
@@ -0,0 +1,117 @@
+//import java.util.Scanner;
+//
+//public class ChatBot {
+//    private static final Scanner in = new Scanner(System.in);
+//
+//    private static final String DIVIDER = "____________________________________________________________";
+//    private static final int TASK_SIZE = 100;
+//    private static final String MESSAGE_WELCOME = "How can I assist you? Type something below! :D";
+//
+//    /**
+//     * Prints a divider for ChatBot
+//     */
+//    private void printDivider() {
+//        System.out.println(DIVIDER);
+//    }
+//
+//    private void startChat() {
+//
+//    }
+//
+//    /**
+//     * Begins the task state until exit is called
+//     */
+//    public void startToDo() {
```

Comment:
Isn't this method a little long? You could add a few more methods within this one to make your code more readable.